AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -269,6 +269,8 @@ codesandbox.io,damota.me,pracazarohom.*,seduo.*#%#//scriptlet('remove-class', 'f
 sportiva.shueisha.co.jp#%#//scriptlet('trusted-click-element', '#onetrust-accept-btn-handler')
 mypalmshop.de#%#//scriptlet('set-constant', 'eCookiePopupDialog', 'emptyObj')
 pca.st##.custom-cookie-banner
+acquisa.de#$#body { overflow: auto !important; }
+acquisa.de#$#.overlayer { display: none !important; }
```

Comment:
Please move these rules to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

Reply:
@AdamWr done